pt/lessons/specifics/mnesia.md

```diff
@@ -0,0 +1,185 @@
+---
+layout: page
+title: Mnesia
+category: specifics
+order: 5
+lang: pt
+---
+
+Mnesia é um sistema de gestão de banco de dados em tempo real de alto tráfego.
+
+## Sumário
+
+- [Visão Geral](#visao-geral)
+- [Quando Usar](#quando-usar)
+- [Schema](#schema)
+- [Nós](#nos)
+- [Iniciando Mnesia](#iniciando-mnesia)
+- [Criando Tabelas](#criando-tabelas)
+- [A Forma Suja](#a-forma-suja)
+- [Operações](#operacoes)
+
+
+## <a name="visao-geral"></a> Visão Geral
+
+Mnesia é um sistema de banco de dados (DBMS) que vem acompanhado da Runtime do Erlang, e por isso podemos utilizar naturalmente com Elixir. O *modelo de dados híbrido relacional e de objeto* do Mnesia é o que o torna adequado para o desenvolvimento de aplicações distribuídas de qualquer escala.
```

Comment:
Change `o torna-o` to either `o torna` or `torna-o` (I think `o torna` is best here)